frontend/src/components/inputs/EntityForm.vue

```diff
@@ -110,6 +124,14 @@ export default {
     batchMode: {
       type: Boolean,
       default: false
+    },
+    attributes: {
+      type: Object,
+      required: false,
+    },
+    id: {
```

Comment:
This is something that only an external component needs. imho it shouldn't be here, but if that's the case maybe at least a comment with where/why it is needed could be helpful? This will be really hard to understand later (especially without TS).